docs/components/sensors/nearfield.md

```diff
@@ -1,3 +1,20 @@
 # Near Field
 
-{!includes/sip.md!}
+Non-visible component to provide NFC capabilities. For now this component supports the reading and writing of text tags only (if supported by the device).  
+
+In order to read and write text tags, the component must have its ReadMode property set to True or False respectively.  
+
+Note: This component will only work on Screen1 of any App Inventor app.
+
+## Properties
+`LastMessage`  
+`ReadMode`  
+`TextToWrite`    
+`WriteType`
+## Event  
+`TagRead(text message)`  
+  Indicates that a new tag has been detected. Currently this is only a plain text tag, as specified in the manifest. See Compiler.java.  
```

Comment:
The note about seeing Compiler.java is a bit irrelevant here, so can you edit that part.
Also, it'd be great if you could add descriptions to the properties.

Reply:
note about compiler.java is fixed and I will add a description of properties, I am working on it